### PR TITLE
Update .NET Core Linux Prerequisites for Alpine Linux and RHEL

### PR DESCRIPTION
**Pull request by JRAlexander**
## Update .NET Core Linux Prerequisites for Alpine Linux and RHEL
Also for latest .NET Core 2.0 SDK and Runtimes

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?branch=pr-en-us-5758)

### CODE REVIEW

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -23,17 +23,39 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 NET Core 2.x is supported on the following Linux 64-bit (`x86_64` or `amd64`) distributions/versions:
 
+**NET Core 2.1**
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: remove line 24. it reads a bit repetitive

File: docs/core/linux-prerequisites.md [hunk 2]
```diff
@@ -23,17 +23,39 @@ This article shows the dependencies needed to develop .NET Core applications on
...
+* Oracle Linux 7
+* Fedora 27
+* Debian 9, 8.7 or later versions
+* Ubuntu 18.04, 17.10, 16.04, 14.04
+* Linux Mint 18, 17
+* openSUSE 42.3 or later versions
+* SUSE Enterprise Linux (SLES) 12 Service Pack 2 or later
+* Alpine Linux 3.7 or later versions
+
+See [.NET Core 2.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for the complete list of .NET Core 2.0 supported operating systems, distributions and versions, out of support OS versions, and lifecycle policy links.
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: link to 2.1 here instead? https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md

File: docs/core/linux-prerequisites.md [hunk 3]
```diff
@@ -114,7 +136,19 @@ To install .NET Core on supported RHEL versions:
...
-To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-current) for supported RHEL versions.
+**.NET Core 2.0**
+
+ Install .NET Core 2.x on supported RHEL versions:
+
+* .NET Core Runtime 2.0.8 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.8)
+* .NET Core Runtime 2.0.7 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.7)
+* .NET Core Runtime 2.0.6 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.6)
+* .NET Core Runtime 2.0.5 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.5)
+* .NET Core SDK 2.1.200 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.1.200)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: These are all still 2.0 SDKs. Might want to call them out like they do in the .net site to make it less confusing (screenshot from https://www.microsoft.com/net/download/all):

image

File: docs/core/linux-prerequisites.md [hunk 4]
```diff
@@ -114,7 +136,19 @@ To install .NET Core on supported RHEL versions:
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-current) for supported RHEL versions.
+**.NET Core 2.0**
+
+ Install .NET Core 2.x on supported RHEL versions:
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: 2.x or 2.0? I'm not seeing the 2.1 links here

File: docs/core/linux-prerequisites.md [hunk 5]
```diff
@@ -114,7 +136,19 @@ To install .NET Core on supported RHEL versions:
...
+
+* .NET Core Runtime 2.0.8 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.8)
+* .NET Core Runtime 2.0.7 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.7)
+* .NET Core Runtime 2.0.6 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.6)
+* .NET Core Runtime 2.0.5 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.5)
+* .NET Core SDK 2.1.200 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.1.200)
+* .NET Core SDK 2.1.105 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.1.105)
+* .NET Core SDK 2.1.103 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.1.103)
+* .NET Core SDK 2.0.3 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.0.3)
+* .NET Core SDK 2.0.0 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.0.0)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: What about 2.1 downloads?

**Review by mairaw — APPROVED** [review]
LGTM but left some comments in case you want to address them

File: docs/core/linux-prerequisites.md (hunk 4 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: 2.0

File: docs/core/linux-prerequisites.md (hunk 5 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Not yet

**Pull request merged**